modules/sdk-coin-ada/src/transactionBuilderFactory.ts

```diff
@@ -11,8 +11,21 @@ export class TransactionBuilderFactory extends BaseTransactionBuilderFactory {
   }
 
   /** @inheritdoc */
-  from(raw: Uint8Array | string): TransactionBuilder {
-    throw new NotImplementedError('from not implemented');
+  from(raw: string): TransactionBuilder {
```

Comment:
Consider adding BREAKING CHANGE or exclamation marks after commit type as this changes how function parameters are passed in

Reply:
Side note, BitGoJS fails the `!`, so we have to use BREAKING CHANGE in the commit footer.

Reply:
Ah good to know